ui/pages/home/home.component.js

```diff
@@ -756,6 +762,98 @@ export default class Home extends PureComponent {
     );
   }
 
+  renderOnboardingPopover = () => {
+    const { t } = this.context;
+    const { setDataCollectionForMarketing } = this.props;
+
+    return (
+      <Popover
```

Comment:
Can we use the new DS Popover component here?

Reply:
https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-popover--docs